src/sentry/digests/utilities.py

```diff
@@ -23,3 +24,96 @@ def get_digest_metadata(digest):
                     end = record.datetime
 
     return start, end, counts
+
+
+def get_personalized_digests(project_id, digest, user_ids):
+    """
+    get_personalized_digests(project: Project, digest: Digest, users: Set[User]) -> Iterator[User, Digest]
+    """
+    # TODO(LB): I Know this is inefficent.
+    if not ProjectOwnership.objects.filter(project_id=project_id).exists():
+        return ProjectOwnership.Everyone
+    records = get_records_from_digests(digest)
```

Comment:
I don't appear to ever use records. This should probably be get_events_from_digest

Reply:
That seems like a reasonable change to me.